### ISSUE TITLE
How to modify batchnormalization to normalize with zero mean?

### ISSUE THREAD

**Issue by HyunkyuPark-biorobotics** [keras-team/keras#11712]
Dear keras users,

I want to modify the batchnormalization layer to the layer that uses moving mean of zero.

At the source code, 

I changed the below code, from the def build, to 'self.moving_mean=None'

self.moving_mean = self.add_weight(
            shape=shape,
            name='moving_mean',
            initializer=self.moving_mean_initializer,
            trainable=False)

But it did not work as I expected.

Can you help me to solve the problem?

Thank you so much.

Best regards
